visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -248,9 +253,24 @@ export class CodeMapMouseEventService
 		return labelForBuilding
 	}
 
+	private EnableOrbitalsRotation(isRotation: boolean) {
+		this.threeViewerService.enableRotation(isRotation)
+		this.viewCubeMouseEventsService.enableRotation(isRotation)
+	}
+
+	onDocumentMouseEnter() {
+		this.EnableOrbitalsRotation(true)
```

Comment:
I am not sure about the code style. I assume that functions/methods should be written camelCase.

Reply:
oops :)